### PR TITLE
Add metrics for HttpHealthCheckedEndpointGroup

### PR DESCRIPTION
**Pull request by imasahiro**

### CODE REVIEW

**Comment by imasahiro** [comment]
Updated

File: core/src/main/java/com/linecorp/armeria/client/endpoint/healthcheck/HealthCheckedEndpointGroup.java [hunk 1]
```diff
@@ -158,14 +167,22 @@ public String toString() {
...
             this.healthChecker = healthChecker;
+            metricRegistry.register(MetricRegistry.name("health-check", endpoint.authority(), "healthy"),
+                                    (Gauge<Integer>) () -> healthy ? 1 : 0);
         }
 
         Endpoint endpoint() {
             return endpoint;
         }
+
+        void healthy(boolean healthy) {
```
Comment (trustin) [inline core/src/main/java/com/linecorp/armeria/client/endpoint/healthcheck/HealthCheckedEndpointGroup.java]: nit: setHealthy?

File: core/src/test/java/com/linecorp/armeria/client/endpoint/healthcheck/HttpHealthCheckedEndpointGroupTest.java [hunk 2]
```diff
@@ -4,6 +4,8 @@
```
Comment (trustin) [inline core/src/test/java/com/linecorp/armeria/client/endpoint/healthcheck/HttpHealthCheckedEndpointGroupTest.java]: Sorry, but could you add the copyright header here as well? (You forgot to add one in your last PR ;-))

Comment (imasahiro) [inline core/src/test/java/com/linecorp/armeria/client/endpoint/healthcheck/HttpHealthCheckedEndpointGroupTest.java]: 💣 Added

Comment (trustin) [inline core/src/test/java/com/linecorp/armeria/client/endpoint/healthcheck/HttpHealthCheckedEndpointGroupTest.java]: Thanks!

**Comment by imasahiro** [comment]
Thanks @trustin !

**Pull request merged**